### ISSUE TITLE
gglagplot shows no points

### ISSUE THREAD

**Issue by robjhyndman** [robjhyndman/forecast#317]
```
gglagplot(woolyrnq, lags=9, do.lines=FALSE, labels=FALSE)
```
I was expecting this to show unlabelled points rather than nothing.

Also
```
gglagplot(woolyrnq, lags=9, do.lines=FALSE, labels=FALSE) + geom_point()
```
is very slow and has huge ugly points.

**Comment by mitchelloharawild** [comment]
Oh, the points. I'll add that now.

**Comment by robjhyndman** [comment]
Please add + geom_point() when do.lines=FALSE and labels=FALSE.
Also make labels=FALSE by default (rather than !do.lines).

**Comment by mitchelloharawild** [comment]
Fixed in #317
